### PR TITLE
hardening/893_add_tests_for_admin_methods

### PR DESCRIPTION
**Pull request by pcoello25**
* Partially fix issue #893 
  * Update CHANGES_NEXT_RELEASE is not neccesary
* 100% unit tests passed:
```
Results : Tests run: 72, Failures: 0, Errors: 0, Skipped: 0
```
* Test logs:
```
Running com.telefonica.iot.cygnus.management.ManagementInterfaceTest
[ManagementInterface.handlePostOneAgentConfParam] --- - POST method post a single parameter in an agent configuration file'.
[ManagementInterface.handlePostOneAgentConfParam] --- -  OK  - Agent configuration parameter posted
[ManagementInterface.handlePutOneAgentConfParam] ---- - PUT method puts a single parameter in an agent configuration file'.
[ManagementInterface.handlePutOneAgentConfParam] ---- -  OK  - Agent configuration parameter put
[ManagementInterface.handleDeleteOneAgentConfParam] - - DELETE method deletes a single parameter in an agent configuration file'.
[ManagementInterface.handleDeleteOneAgentConfParam] - -  OK  - Agent configuration parameter deleted
[ManagementInterface.handleGetAgentConfParams] ------ - GET method gets all agent configuration parameters'.
[ManagementInterface.handleGetAgentConfParams] ------ -  OK  - All agent configuration parameters obtained
[ManagementInterface.handleGetOneAgentConfParam] ---- - GET method gets one agent configuration parameter'.
[ManagementInterface.handleGetOneAgentConfParam] ---- -  OK  - Agent configuration parameter obtained
```
* Assignee: @frbattid

### CODE REVIEW

File: cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java [hunk 1]
```diff
@@ -622,5 +644,256 @@ public void testGetMethodsGetsAllSubscriptionsV2() throws Exception {
...
+     /**
+     * [ManagementInterface] -------- 'GET method gets parameters of a given agent configuration file'.
+     * @throws java.lang.Exception
+     */
+    @Test
+    public void testGetMethodGetsAllAgentConfigurationParameters() throws Exception {
+        System.out.println(getTestTraceHead("[ManagementInterface.handleGetAgentConfParams]") + " - GET "
+                + "method gets all agent configuration parameters'.");
+     
+         File file;
```
Comment (frbattid) [inline cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java]: This seems to be bad indented

File: cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java [hunk 2]
```diff
@@ -622,5 +644,256 @@ public void testGetMethodsGetsAllSubscriptionsV2() throws Exception {
...
+    } // testPostMethodPostOneAgentConfigurationParameter
+    
+    /**
+     * [ManagementInterface] -------- 'PUT method puts a single parameter in a given agent configuration file'.
+     * @throws java.lang.Exception
+     */
+    @Test
+    public void testPutMethodPutOneAgentConfigurationParameter() throws Exception {
+        System.out.println(getTestTraceHead("[ManagementInterface.handlePutOneAgentConfParam]") + " - PUT "
+                + "method puts a single parameter in an agent configuration file'.");
```
Comment (frbattid) [inline cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java]: Final ' is not necessary

File: cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java [hunk 3]
```diff
@@ -622,5 +644,256 @@ public void testGetMethodsGetsAllSubscriptionsV2() throws Exception {
...
      } // testGetMethodsGetsAllSubscriptionsV2
+     
+     /**
+     * [ManagementInterface] -------- 'GET method gets parameters of a given agent configuration file'.
+     * @throws java.lang.Exception
+     */
+    @Test
+    public void testGetMethodGetsAllAgentConfigurationParameters() throws Exception {
+        System.out.println(getTestTraceHead("[ManagementInterface.handleGetAgentConfParams]") + " - GET "
+                + "method gets all agent configuration parameters'.");
```
Comment (frbattid) [inline cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java]: Final ' is not necessary

File: cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java [hunk 4]
```diff
@@ -622,5 +644,256 @@ public void testGetMethodsGetsAllSubscriptionsV2() throws Exception {
...
+    } // testGetMethodGetsOneAgentConfigurationParameter
+    
+    /**
+     * [ManagementInterface] -------- 'POST method posts a single parameter in a given agent configuration file'.
+     * @throws java.lang.Exception
+     */
+    @Test
+    public void testPostMethodPostOneAgentConfigurationParameter() throws Exception {
+        System.out.println(getTestTraceHead("[ManagementInterface.handlePostOneAgentConfParam]") + " - POST "
+                + "method post a single parameter in an agent configuration file'.");
```
Comment (frbattid) [inline cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java]: Final ' is not necessary

File: cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java (hunk 1 above)
Comment (pcoello25) [inline cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java]: Fixed in 3a9c4dd (All this method was reordered but the change is fixed)

File: cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java (hunk 4 above)
Comment (pcoello25) [inline cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java]: Fixed in 7d2245e

File: cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java (hunk 3 above)
Comment (pcoello25) [inline cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java]: Fixed in 7d2245e

File: cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java (hunk 2 above)
Comment (pcoello25) [inline cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java]: Fixed in 7d2245e

**Comment by frbattid** [comment]
LGTM

**Pull request merged**